### ISSUE TITLE
Table is no longer usable for screen reader users with Jaws 2026

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#946]
Summary

Table is no longer usable for screen reader users with jaws 2026 which was working perfectly fine with Jaws 2025

Example:

Attached the snippix file

[JAWS regression missing announcements for grid.html](https://github.com/user-attachments/files/29037226/JAWS.regression.missing.announcements.for.grid.html)
[JAWS regression missing announcements for table.html](https://github.com/user-attachments/files/29037225/JAWS.regression.missing.announcements.for.table.html)

Issue:

We recently found a couple of regression issues in JAWS that heavily impact the accessibility of the tables. I still haven't confirmed them with Stefan, but I will do as soon as he is back from vacation. Due to the severity of the issues and as discussed with Nikolay we would like to report them to Freedom Scientific with high priority and as soon as possible.

In the past we did quite some effort to get rid of custom announcements in the table and we currently rely on JAWS announcements for the row and column count as well as the column headers. This was working perfectly with JAWS 2025, no missing or double announcements. Unfortunately JAWS 2026 doesn't announce these information anymore, just the cell content. With that the table is not usable for JAWS users. We haven't done any related changes meanwhile and NVDA announces the information as expected. Please find attached a plain html example for testing and for the report.

**Comment by JeremiahRogersVispero** [comment]
I see you closed this issue, @havvamehfooz. Thanks for your work and the post. For others who might encounter this issue later, I presume the difference @havvamehfooz refers to between JAWS 2025 and JAWS 2026 is the information JAWS announces when pressing TAB or SHIFT+TAB to navigate from one focusable table cell to another. This change was intentional.
